learn/platform/networks/moonriver.md

```diff
@@ -15,25 +15,25 @@ In order to collect as much feedback as possible and provide fast issue resoluti
 
 ## Initial Configurations {: #initial-configurations } 
 
-Moonriver has the following configurations:
+Currently, Moonriver has the following configurations:
 
 - Runs as a parachain connected to the Kusama relay chain
-- Has an active set of {{ networks.moonriver.staking.max_candidates }} collators
-- There are two RPC endpoints (hosted by PureStake). People can run full nodes to access their own private RPC endpoints
+- Has an active set of {{ networks.moonriver.staking.max_collators }} collators
```

Comment:
I see you changed this back to max_collators, but the variables.yml file uses max_candidates for Moonbase Alpha and Moonriver. I suggest that we be consistent, so if you want to use max_collators for Moonbeam, you should also use it for Moonriver and Moonbase and update all instances throughout the code base.

Reply:
Good point! I've fixed this